tests/Test/BlackBoxPushGatewayTest.php

```diff
@@ -10,22 +10,29 @@
 
 class BlackBoxPushGatewayTest extends TestCase
 {
+    public function transportProvider()
+    {
+        yield ['http'];
+        yield ['https'];
+    }
+
```

Comment:
Why are you using yield here?

Reply:
Data providers do not indeed need an array as a return, but an iterator can fit better for making them a little more readable.